invenio_curations/assets/semantic-ui/js/invenio_curations/deposit/CurationRequestForm.js

```diff
@@ -139,7 +139,7 @@ export class CurationsContainerComponent extends Component {
                   {latestRequest ? (
                     <>
                       <Label verticalAlign="top">{i18next.t("Status")}</Label>
-                      <RequestStatusLabel status={latestRequest.status} />
+                      <RequestStatusLabel status={i18next.t(latestRequest.status)} />
```

Comment:
does this work with translations or would we need to add additional components for each translation?
i.e.:
- English: `"RequestStatusLabel.layout.critiqued": LabelStatusCritique,`
- German `"RequestStatusLabel.layout.kritisiert": LabelStatusCritique,`

I would assume `LabelStatusCritique` takes care of translating the text to display. Might be wrong here, so please correct me if this is wrong :smile: 
Will be able to test this next week just to be sure and have clarity

Reply:
You are absolutely right, this change made matters worse!
The `RequestStatusLabel` needs the "raw" value for the status, and actually uses it to create a button on the fly.